t/descriptive.t

```diff
@@ -451,19 +451,38 @@ EOO
     );
   }
 
-  if (@warnings == 1) {
-    pass("got one warning about ambiguity of options");
-    like(
-      $warnings[0],
-      qr/these ambiguous options: f/,
-      "GLD warns on ambiguity for you",
-    );
-  } elsif (! @warnings) {
-    fail("got one warning about ambiguity of options");
-    diag("expected a warning but got none");
-  } else {
-    fail("got one warning about ambiguity of options");
-    diag("warning: $_") for @warnings;
+  note("Using Getopt::Long version $Getopt::Long::VERSION");
+  if ($Getopt::Long::VERSION < 2.55) {
+    if (@warnings == 1) {
```

Comment:
I think this whole patch could be shortened to:
```perl
    if (@warnings == 1 + (Getopt::Long->VERSION >= 2.55)) {
```
But that may be a bit too cheeky.

Reply:
It *is* cheeky -- but I would expect nothing less of @mauke! ;-)

However, then the test descriptions would become a bit misleading, *i.e.*, `one warning` versus `two warnings`.